drivers/gpio/gpio_nrf5.c

```diff
@@ -375,6 +383,28 @@ static const struct gpio_driver_api gpio_nrf5_drv_api_funcs = {
 	.disable_callback = gpio_nrf5_disable_callback,
 };
 
+/* Enable GPIOTE Interrupt */
+void nrf_gpiote_interrupt_enable(uint32_t mask)
+{
+	NRF_GPIOTE->EVENTS_PORT = 0;
+	nrf_gpiote_int_enable(mask);
+}
+
+/* Disable GPIOTE Interrupt */
+void  nrf_gpiote_interrupt_disable(uint32_t mask)
```

Comment:
There is no reference to this function, delete it.

Reply:
as we have option to Enable GPIOTE interrupt, I want to put for Disable too for completeness incase someone want to use it. 
If we feel it is not needed, i can put another patch to delete it.